docs/get-started/allocation/allocation-change-request.md

```diff
@@ -82,4 +104,18 @@ This will show more details about the change request as shown below:
 
 ![Allocation Change Request Details for OpenShift Project](images/coldfront-openshift-change-requested-details.png)
```

Comment:
It might help to have values in the red area. For example the resources required for a A100SXM4 flavor.

Reply:
I added a separate "tips" section on how to choose those quota values. We would not edit this generic screenshot on how to specify the quota values that can be based on user's requirements.